.github/workflows/build-test.yml

```
@@ -0,0 +1,54 @@
+name: Build, Lint, and Test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  build-test:
+    name: Build, and Test
+    runs-on: ubuntu-20.04
```

Consider an upgrade:
```suggestion
    runs-on: ubuntu-latest
```